payments-core/src/main/java/com/stripe/android/payments/bankaccount/navigation/CollectBankAccountContract.kt

```diff
@@ -32,26 +34,32 @@ internal class CollectBankAccountContract :
         )
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     sealed class Args(
         open val publishableKey: String,
         open val clientSecret: String,
         open val configuration: CollectBankAccountConfiguration,
+        open val attachToIntent: Boolean
```

Comment:
Since what this does is not very obvious, let's add a brief comment explaining what it does maybe?